### ISSUE TITLE
Sentinel config file conflict

### ISSUE THREAD

**Issue by boxidau** [rackspace-cookbooks/redis-multi#7]
Sentinel actually writes runtime information to it's config file which is subsequently overwritten by chef

I propose that we use a "fixed" config file which is managed by chef and a runtime copy of the config which is not chef managed and is used as the source of truth for sentinel.

On any updates to the chef-managed config the sentinel runtime config is also overwritten with the updated chef content and sentinel is reloaded. Otherwise if no changes are made to the chef-managed config the runtime file should just be checked for existence (and copied from the chef managed version if it doesn't exist).

**Comment by boxidau** [comment]
http://code.flickr.net/2014/07/31/redis-sentinel-at-flickr/

**Comment by boxidau** [comment]
This probably needs to go upstream

**Comment by martinb3** [comment]
Yes, this definitely needs to go upstream @ [the redisio cookbook](https://github.com/brianbianco/redisio). Upstream still has a few different pieces of sentinel work to do (e.g. support for multiple masters). I'd be much obliged if you would submit an issue there.

**Comment by brint** [comment]
Redis v3.0.0 has just been released with native clustering support. We will need to re-evaluate how we handle this based on the new version.
